de.variantsync.core/src/main/java/de/variantsync/core/ast/JsonParserASTwithLineGrammar.java

```diff
@@ -0,0 +1,73 @@
+package de.variantsync.core.ast;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import de.variantsync.core.interfaces.Grammar;
+
+/**
+ * 
+ * Uses gson to import and export ASTs to and from json. A generic use of grammar is not possible, hence for each grammar a different class has to be created.
+ * In this case the used grammar is LineGrammar.
+ * 
+ * @author jerem
+ *
+ */
+public class JsonParserASTwithLineGrammar {
+
+	static Gson gson = new GsonBuilder().setPrettyPrinting().excludeFieldsWithoutExposeAnnotation().create();
```

Comment:
Either document why you constructed the Gson like that with a comment or more descriptive variable name than just `gson`.